xml/obs_build_containers.xml

```diff
@@ -14,34 +14,36 @@
    <remark>TBD</remark>
   </abstract>
  </info>
- <para> Containers are workloads which container all necessary files to make
+ <para> Containers are workloads which embed all necessary files to make
   the workload independent of the running host OS. This includes (but is not
   limited to) libraries, executables and shared resource files. </para>
 
  <sect1>
```

Comment:
I'd suggest adding a `xml:id=...` attribute to all sections, for easier referencing from other parts of the book (and for use as permalinks to a particular section.

Reply:
I've added those as well.